src/main/java/org/zalando/nakadi/service/subscription/SubscriptionService.java

```diff
@@ -143,21 +143,21 @@ public Subscription createSubscription(final SubscriptionBase subscriptionBase)
         return subscription;
     }
 
-    private void checkFeatureToggles(final SubscriptionBase subscription) {
+    private void checkFeatureTogglesForSubscription(final SubscriptionBase subscription) {
```

Comment:
Sorry, I still think we need a better name for this. With this name, someone may think that this method also applied to getExistingSubscription(..) or listSubscriptions(..).

I would suggest: `checkFeatureTogglesForCreationAndUpdate(..)`. It's a mouthful, but it's clear.